devnet-sdk/system/chain.go

```diff
@@ -48,11 +55,51 @@ func (m *clientManager) Client(rpcURL string) (*ethclient.Client, error) {
 		return client, nil
 	}
 
+	ethClCfg := sources.EthClientConfig{
+		MaxRequestsPerBatch:   10,
+		MaxConcurrentRequests: 10,
+		ReceiptsCacheSize:     10,
+		TransactionsCacheSize: 10,
+		HeadersCacheSize:      10,
+		PayloadsCacheSize:     10,
+		TrustRPC:              false,
+		MustBePostMerge:       true,
+		RPCProviderKind:       sources.RPCKindStandard,
+		MethodResetDuration:   time.Minute,
+	}
+	rpcClient, err := rpc.DialContext(context.Background(), rpcURL)
+	if err != nil {
+		return nil, err
+	}
+	ethCl, err := sources.NewEthClient(client.NewBaseRPCClient(rpcClient), log.Root(), nil, &ethClCfg)
+	if err != nil {
+		return nil, err
+	}
+	m.clients[rpcURL] = ethCl
+	return ethCl, nil
+}
+
+func (m *clientManager) LowLevelClient(rpcURL string) (*ethclient.Client, error) {
```

Comment:
nit: GethClient might be a good name since it is not relative to any particular frame of reference. From the perspective of ethclient.Client, BaseRPCClient is a low-level client.

Reply:
Fixed at https://github.com/ethereum-optimism/optimism/pull/14751/commits/8863173d6d0fc496c6774e808d8079afc0d761bf and /pull/14751/commits/604467515439e3f7c6779e51a5ee9498c0fefa65